src/DataMapper/PropertyMapper.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the RichModelFormsBundle package.
+ *
+ * (c) Christian Flothmann <christian.flothmann@sensiolabs.de>
+ * (c) Christopher Hertel <christopher.hertel@sensiolabs.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types = 1);
+
+namespace SensioLabs\RichModelForms\DataMapper;
+
+/**
+ * @author Christian Flothmann <christian.flothmann@sensiolabs.de>
+ */
+interface PropertyMapper
```

Comment:
IMO we should switch to using the `Interface` suffix like symfony does
might be sensible as follow up PR to rename `ExceptionHandler` as well

Reply:
see also #52